docs/DeveloperGuide.md

```diff
@@ -61,9 +61,9 @@ The *Sequence Diagram* below shows how the components interact with each other f
 Each of the four main components (also shown in the diagram above),
 
 * defines its *API* in an `interface` with the same name as the Component.
-* implements its functionality using a concrete `{Component Name}Manager` class (which follows the corresponding API `interface` mentioned in the previous point.
+* implements its functionality using a concrete `{Component Name} Manager` class (which follows the corresponding API `interface` mentioned in the previous point).
```

Comment:
I think original version is correct. the aforementioned classes follow the format `*Manager` without a space e.g. `LogicManager.java`, not `Logic Manager.java`

Reply:
Ah okie thank you, will update.